docs/static/api-docs/slack_sdk/audit_logs/index.html

```diff
@@ -420,39 +420,39 @@ <h3>Class variables</h3>
 <dl>
 <dt id="slack_sdk.audit_logs.AuditLogsClient.BASE_URL"><code class="name">var <span class="ident">BASE_URL</span></code></dt>
 <dd>
-<div class="desc"></div>
+<div class="desc"><p>The type of the None singleton.</p></div>
```

Comment:
Just curious- are most of the references due to a more recent `python` version being used? I'm a fan of the detail 📚 ✨

Reply:
I'm assuming it is, I used the latests version of python to generate the docs 🤔